WhatsApp/app/src/main/res/values/themes.xml

```diff
@@ -14,10 +14,13 @@
         <item name="actionModeBackground">@color/teal_green</item>
 
         <item name="windowActionModeOverlay">true</item>
-
+        <item name="android:forceDarkAllowed" tools:targetApi="q">false</item>
```

Comment:
why are we using tools here

Reply:
was just testing all available tools for changing themes, will remove it now.